2022-04-09 16:14:21,829 INFO    wandb_internal:8815 [internal.py:wandb_internal():91] W&B internal server running at pid: 8815, started at: 2022-04-09 16:14:21.828726
2022-04-09 16:14:21,829 INFO    MainThread:8815 [backend.py:ensure_launched():137] started backend process with pid: 0
2022-04-09 16:14:21,830 INFO    MainThread:8815 [wandb_init.py:init():423] backend started and connected
2022-04-09 16:14:21,831 DEBUG   MainThread:8815 [config_util.py:dict_from_config_file():101] no default config file found in config-defaults.yaml
2022-04-09 16:14:21,834 INFO    MainThread:8815 [wandb_init.py:init():465] updated telemetry
2022-04-09 16:14:21,834 INFO    MainThread:8815 [wandb_init.py:init():484] communicating current version
2022-04-09 16:14:21,835 DEBUG   HandlerThread:8815 [handler.py:handle_request():124] handle_request: check_version
2022-04-09 16:14:21,835 INFO    WriterThread:8815 [datastore.py:open_for_write():77] open: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/run-3t82t88x.wandb
2022-04-09 16:14:21,835 DEBUG   SenderThread:8815 [sender.py:send():179] send: header
2022-04-09 16:14:21,835 DEBUG   SenderThread:8815 [sender.py:send_request():193] send_request: check_version
2022-04-09 16:14:21,935 INFO    MainThread:8815 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.14 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-04-09 16:14:21,936 INFO    MainThread:8815 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-04-09 16:14:21,939 DEBUG   SenderThread:8815 [sender.py:send():179] send: run
2022-04-09 16:14:23,089 INFO    SenderThread:8815 [dir_watcher.py:__init__():168] watching files in: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files
2022-04-09 16:14:23,089 INFO    SenderThread:8815 [sender.py:_start_run_threads():707] run started: 3t82t88x with start time 1649501061
2022-04-09 16:14:23,090 DEBUG   SenderThread:8815 [sender.py:send():179] send: summary
2022-04-09 16:14:23,091 INFO    SenderThread:8815 [sender.py:_save_file():829] saving file wandb-summary.json with policy end
2022-04-09 16:14:23,091 INFO    MainThread:8815 [wandb_init.py:init():522] starting run threads in backend
2022-04-09 16:14:23,092 DEBUG   HandlerThread:8815 [handler.py:handle_request():124] handle_request: run_start
2022-04-09 16:14:23,094 DEBUG   HandlerThread:8815 [meta.py:__init__():39] meta init
2022-04-09 16:14:23,094 DEBUG   HandlerThread:8815 [meta.py:__init__():53] meta init done
2022-04-09 16:14:23,094 DEBUG   HandlerThread:8815 [meta.py:probe():210] probe
2022-04-09 16:14:23,100 DEBUG   HandlerThread:8815 [meta.py:_setup_git():200] setup git
2022-04-09 16:14:23,122 DEBUG   HandlerThread:8815 [meta.py:_setup_git():207] setup git done
2022-04-09 16:14:23,122 DEBUG   HandlerThread:8815 [meta.py:_save_code():89] save code
2022-04-09 16:14:23,133 DEBUG   HandlerThread:8815 [meta.py:_save_code():110] save code done
2022-04-09 16:14:23,133 DEBUG   HandlerThread:8815 [meta.py:_save_patches():127] save patches
2022-04-09 16:14:23,196 DEBUG   HandlerThread:8815 [meta.py:_save_patches():169] save patches done
2022-04-09 16:14:23,196 DEBUG   HandlerThread:8815 [meta.py:_save_pip():57] save pip
2022-04-09 16:14:23,197 DEBUG   HandlerThread:8815 [meta.py:_save_pip():71] save pip done
2022-04-09 16:14:23,197 DEBUG   HandlerThread:8815 [meta.py:_save_conda():78] save conda
2022-04-09 16:14:24,090 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/code/train_translation.py
2022-04-09 16:14:24,090 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/diff.patch
2022-04-09 16:14:24,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/requirements.txt
2022-04-09 16:14:24,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/conda-environment.yaml
2022-04-09 16:14:24,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/wandb-summary.json
2022-04-09 16:14:24,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/code
2022-04-09 16:14:24,537 DEBUG   HandlerThread:8815 [meta.py:_save_conda():86] save conda done
2022-04-09 16:14:24,538 DEBUG   HandlerThread:8815 [meta.py:probe():252] probe done
2022-04-09 16:14:24,539 DEBUG   SenderThread:8815 [sender.py:send():179] send: files
2022-04-09 16:14:24,539 INFO    SenderThread:8815 [sender.py:_save_file():829] saving file wandb-metadata.json with policy now
2022-04-09 16:14:24,540 INFO    SenderThread:8815 [sender.py:_save_file():829] saving file code/train_translation.py with policy now
2022-04-09 16:14:24,541 INFO    SenderThread:8815 [sender.py:_save_file():829] saving file diff.patch with policy now
2022-04-09 16:14:24,547 INFO    MainThread:8815 [wandb_run.py:_console_start():1538] atexit reg
2022-04-09 16:14:24,548 DEBUG   HandlerThread:8815 [handler.py:handle_request():124] handle_request: stop_status
2022-04-09 16:14:24,548 DEBUG   SenderThread:8815 [sender.py:send_request():193] send_request: stop_status
2022-04-09 16:14:24,551 INFO    MainThread:8815 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2022-04-09 16:14:24,551 INFO    MainThread:8815 [wandb_run.py:_redirect():1449] Wrapping output streams.
2022-04-09 16:14:24,552 INFO    MainThread:8815 [wandb_run.py:_redirect():1473] Redirects installed.
2022-04-09 16:14:24,552 INFO    MainThread:8815 [wandb_init.py:init():547] run started, returning control to user process
2022-04-09 16:14:24,553 INFO    MainThread:8815 [wandb_run.py:_config_callback():787] config_cb None None {'workers': 4, 'epochs': 5, 'batch_size': 4, 'learning_rate': 0.2, 'dropout': 0.01, 'weight_decay': 1e-06, 'clip': 1, 'betas': [0.9, 0.98], 'eps': 1e-09, 'loss_fn': 'cross_entropy', 'dmodel': 768, 'nhead': 4, 'dfeedforward': 256, 'nlayers': 3, 'projector': '768-256', 'tokenizer': 'bert-base-multilingual-uncased', 'mbert_out_size': 768, 'checkpoint_dir': 'checkpoint', 'load': 0, 'checkbleu': 5, 'train': True, 'print_freq': 5, 'test_translation': 1, 'ngpus_per_node': 2, 'rank': 0, 'dist_url': 'tcp://localhost:58472', 'world_size': 2}
2022-04-09 16:14:25,090 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_modified():229] file/dir modified: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/conda-environment.yaml
2022-04-09 16:14:25,093 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/wandb-metadata.json
2022-04-09 16:14:25,093 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_created():216] file/dir created: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log
2022-04-09 16:14:25,577 DEBUG   SenderThread:8815 [sender.py:send():179] send: config
2022-04-09 16:14:26,654 INFO    Thread-14 :8815 [upload_job.py:push():133] Uploaded file /tmp/tmpyo0egpl2wandb/1woflnrf-wandb-metadata.json
2022-04-09 16:14:26,655 INFO    Thread-17 :8815 [upload_job.py:push():133] Uploaded file /tmp/tmpyo0egpl2wandb/2g34m9v2-code/train_translation.py
2022-04-09 16:14:27,090 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_modified():229] file/dir modified: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log
2022-04-09 16:14:27,669 INFO    Thread-18 :8815 [upload_job.py:push():133] Uploaded file /tmp/tmpyo0egpl2wandb/1gwzitp2-diff.patch
2022-04-09 16:14:29,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_modified():229] file/dir modified: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log
2022-04-09 16:14:29,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_modified():229] file/dir modified: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/config.yaml
2022-04-09 16:14:31,091 INFO    Thread-11 :8815 [dir_watcher.py:_on_file_modified():229] file/dir modified: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log
2022-04-09 16:14:40,579 DEBUG   HandlerThread:8815 [handler.py:handle_request():124] handle_request: stop_status
2022-04-09 16:14:40,579 DEBUG   SenderThread:8815 [sender.py:send_request():193] send_request: stop_status
2022-04-09 16:14:51,743 DEBUG   SenderThread:8815 [sender.py:send():179] send: stats
2022-04-09 16:14:56,424 DEBUG   HandlerThread:8815 [handler.py:handle_request():124] handle_request: stop_status
2022-04-09 16:14:56,424 DEBUG   SenderThread:8815 [sender.py:send_request():193] send_request: stop_status
2022-04-09 16:15:01,820 DEBUG   SenderThread:8815 [sender.py:send():179] send: history
2022-04-09 16:15:01,820 INFO    WriterThread:8815 [datastore.py:close():258] close: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/run-3t82t88x.wandb
2022-04-09 16:15:01,820 INFO    SenderThread:8815 [sender.py:finish():933] shutting down sender
2022-04-09 16:15:01,821 INFO    SenderThread:8815 [dir_watcher.py:finish():282] shutting down directory watcher
2022-04-09 16:15:02,097 INFO    SenderThread:8815 [dir_watcher.py:finish():312] scan: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files
2022-04-09 16:15:02,098 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/requirements.txt requirements.txt
2022-04-09 16:15:02,099 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/wandb-metadata.json wandb-metadata.json
2022-04-09 16:15:02,099 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log output.log
2022-04-09 16:15:02,120 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/conda-environment.yaml conda-environment.yaml
2022-04-09 16:15:02,121 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/wandb-summary.json wandb-summary.json
2022-04-09 16:15:02,142 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/config.yaml config.yaml
2022-04-09 16:15:02,153 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/diff.patch diff.patch
2022-04-09 16:15:02,165 INFO    SenderThread:8815 [dir_watcher.py:finish():318] scan save: /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/code/train_translation.py code/train_translation.py
2022-04-09 16:15:02,166 INFO    SenderThread:8815 [file_pusher.py:finish():176] shutting down file pusher
2022-04-09 16:15:02,166 INFO    SenderThread:8815 [file_pusher.py:join():181] waiting for file pusher
2022-04-09 16:15:04,027 INFO    Thread-25 :8815 [upload_job.py:push():133] Uploaded file /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/output.log
2022-04-09 16:15:04,029 INFO    Thread-27 :8815 [upload_job.py:push():133] Uploaded file /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/wandb-summary.json
2022-04-09 16:15:04,030 INFO    Thread-24 :8815 [upload_job.py:push():133] Uploaded file /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/requirements.txt
2022-04-09 16:15:04,034 INFO    Thread-26 :8815 [upload_job.py:push():133] Uploaded file /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/conda-environment.yaml
2022-04-09 16:15:04,036 INFO    Thread-28 :8815 [upload_job.py:push():133] Uploaded file /home/ivlabs/context_enhancement/context_new/context_enhancement/wandb/run-20220409_161421-3t82t88x/files/config.yaml
2022-04-09 16:15:05,015 ERROR   wandb_internal:8815 [internal.py:wandb_internal():159] Thread HandlerThread:
Traceback (most recent call last):
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/site-packages/wandb/sdk/internal/internal_util.py", line 55, in run
    self._run()
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    record = self._input_record_q.get(timeout=1)
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/multiprocessing/queues.py", line 108, in get
    res = self._recv_bytes()
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/multiprocessing/connection.py", line 216, in recv_bytes
    buf = self._recv_bytes(maxlength)
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/multiprocessing/connection.py", line 407, in _recv_bytes
    buf = self._recv(4)
  File "/home/ivlabs/miniconda3/envs/ectc/lib/python3.7/multiprocessing/connection.py", line 383, in _recv
    raise EOFError
EOFError
2022-04-09 16:24:49,089 INFO    MainThread:8815 [wandb_run.py:_atexit_cleanup():1508] got exitcode: 255
2022-04-09 16:24:49,090 INFO    MainThread:8815 [wandb_run.py:_restore():1480] restore
2022-04-09 16:24:49,379 INFO    MainThread:8815 [wandb_run.py:_restore():1480] restore
2022-04-09 16:24:49,381 INFO    MainThread:8815 [internal.py:handle_exit():78] Internal process exited
